src/logic.service.ts

```diff
@@ -117,6 +127,11 @@ export const handlePain013 = async (transaction: Pain013): Promise<any> => {
     throw err;
   }
 
+  transaction.DataCache = {
+    "cdtrAcctId": creditorHash,
```

Comment:
Here you set cdtrAcctId to the hash, instead of the actual id

Reply:
This is now retrieved from Redis/Database